clou.py

```diff
@@ -0,0 +1,125 @@
+# Features: LotFrontage, LotArea, MasVnrArea, BsmtFinSF1, BsmtFinSF2
+
+def feature_extract(dataframe):
+    
+    # Importing packages
+    import pandas as pd
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import seaborn as sns
+    from scipy.stats import skew
+    
+    # Create my dataframe.
+    myDF = pd.DataFrame() # create an empty dataframe to store my features
+    myFeatures = ['SalePrice','BsmtFinSF2', 'BsmtFinSF1', 'MasVnrArea', 'LotArea', 'LotFrontage']
+    for feature in myFeatures:
+        myDF.insert(loc=0,column=feature,value=dataframe[feature]) # insert my features from the master dataframe into my dataframe
+    
```

Comment:
for 13-16:
Have you considered:
myFeatures = ['SalePrice','BsmtFinSF2', 'BsmtFinSF1', 'MasVnrArea', 'LotArea', 'LotFrontage']
myDF = dataframe[myfeatures]
?
Might be a smidge faster.